### ISSUE TITLE
Runtime extension PR #1913 & magic methods

### ISSUE THREAD

**Issue by mbardelmeijer** [twigphp/Twig#2160]
I'm currently experiencing what i believe is a breaking change after upgrading from v1.25.0 to v1.26.0.

Exception that is shown in the application is:
```
An exception has been thrown during the compilation of a template ("Method Illuminate\Auth\AuthManager::user() does not exist") in "layouts/user.twig".
```

I've traced this to the PR #1913, if i replace the `lib/Twig/Node/Expression/Call.php` file with the one from v1.25.0 everything works fine.

At [line 263](https://github.com/twigphp/Twig/pull/1913/files#diff-2089d573c57338c74075cb6b719c3b94R262) a new ReflectionMethod instance is created, but the method doesn't exist (`user` in the `AuthManager` class in this case). This is true, but the method should be proxied [trough a magic `__call` method](https://github.com/illuminate/auth/blob/master/AuthManager.php#L294) to the `guard` where the `user` method resides. This behaviour worked fine in v1.25.0 as it didn't went trough the `reflectCallable` method.

For reference, the `rcrowe/twigbridge` package which includes the extension: https://github.com/rcrowe/TwigBridge/blob/master/src/Extension/Laravel/Auth.php

Have analysed this to the best of my knowledge, but wasn't able to implement a (good) fix.

If you need any more info, please let me know! Thanks.

**Comment by josecanhelp** [comment]
I am also experiencing this issue. Thank you for the explanation @mbardelmeijer.

**Comment by fabpot** [comment]
Can you check #2161 for a fix?

**Comment by josecanhelp** [comment]
@fabpot It's working for me. Thank you for the quick response!

**Comment by mbardelmeijer** [comment]
@fabpot fix from #2161 works 👍
